### PR TITLE
release: Ensure dnf knows builddep

### PR DESCRIPTION
**Pull request by dperpeet**
We need that plugin to install the cockpit build dependencies.

### CODE REVIEW

**Comment by dperpeet** [comment]
I needed to add this to build a new container

File: Makefile [hunk 1]
```diff
@@ -7,8 +7,29 @@ all:
...
+	TAGS=`docker images --format "table {{.Tag}}\t{{.ID}}" | grep $$ID | awk '{print $$1}'`; \
+	if [ `echo "$$TAGS" | wc -w` -ne "2" ]; then \
+		echo "Expected exactly two tags for the image to push: latest and one other"; \
+		exit 1; \
+	fi; \
+	for PUSHTAG in $$TAGS; do \
+		docker push "cockpit/infra-base:$$PUSHTAG"; \
+	done \
+	}
+	@true
```
Comment (martinpitt) [inline Makefile]: OOI, why the { } and `@true`?

File: Makefile [hunk 2]
```diff
@@ -68,6 +88,25 @@ verify-shell: docker-running
...
+		TAGS=`docker images --format "table {{.Tag}}\t{{.ID}}" | grep $$ID | awk '{print $$1}'`; \
+		if [ `echo "$$TAGS" | wc -w` -ne "2" ]; then \
+			echo "Expected exactly two tags for the image to push: latest and one other"; \
+			exit 1; \
+		fi; \
+		for PUSHTAG in $$TAGS; do \
+			docker push "cockpit/infra-verify:$$PUSHTAG"; \
+		done \
+		}
+		@true
```
Comment (martinpitt) [inline Makefile]: So you have push targets for base and verify, why not for release?

File: Makefile (hunk 1 above)
Comment (dperpeet) [inline Makefile]: I think this was necessary for the ID variable and script to work

File: Makefile (hunk 2 above)
Comment (dperpeet) [inline Makefile]: this code is copy/pasted from the `release-push` target

**Comment by martinpitt** [comment]
Added commit to install rpm-build, which broke the current release attempt:
```
> Starting: release-srpm
rpm: /usr/bin/rpmspec: No such file or directory
release-srpm: could not calculate release number from spec file: tools/cockpit.spec
```

**Comment by dperpeet** [comment]
> Added commit to install rpm-build, which broke the current release attempt:

looks good!

**Comment by martinpitt** [comment]
The 139 release succeeded with these changes, so merging now.

**Pull request merged**